synapse/module_api/__init__.py

```diff
@@ -1452,6 +1452,57 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def create_room(
+        self,
+        user_id: str,
+        config: JsonDict,
+        ratelimit: bool = True,
+        creator_join_profile: Optional[JsonDict] = None,
+    ) -> Tuple[str, Optional[str]]:
+        """Creates a new room.
+
+        Added in Synapse v1.66.0.
+
+        Args:
+            user_id:
+                The user who requested the room creation.
+            config : A dict of configuration options. See "Request body" of:
+                https://spec.matrix.org/latest/client-server-api/#post_matrixclientv3createroom
+            ratelimit: set to False to disable the rate limiter.
```

Comment:
```suggestion
            ratelimit: set to False to disable the rate limiter for this specific join.
```

(as opposed to disabling the rate limiter for any room join)

Reply:
Done here: f7df217